docs/content/stack/zero-configuration.md

```diff
@@ -25,5 +25,7 @@ DOCKSAL_STACK="acquia"
 The following stacks are available:
 
 - `default` - web, db, cli (assumed, when none specified)
+- `default-nodb` - web, cli
 - `acquia` - web, db, cli, varnish, memcached, solr (used specifically for [Acquia](https://www.acquia.com/) hosted projects)
```

Comment:
Let's add the same details for the acquia stack.

```suggestion
 - `acquia` - web (Apache), db (MySQL), cli, varnish, memcached, solr (used specifically for [Acquia](https://www.acquia.com/) hosted projects)
```

Reply:
I love this idea!